aten/src/ATen/gen.py

```diff
@@ -323,6 +366,10 @@ def iterate_types():
         yield (backend, 'Dense')
 
 
+def gen_per_op_registration_filename(opname):
+    return 'pt_op_register_{}.cpp'.format(opname.replace(':', '_'))
```

Comment:
underscores are valid in op names, this could cause issues. Can we use a different character here that is valid for file names but not in op names?

Reply:
This can only cause problem when there are two op names having same filename after the replacement, e.g.: one is "aten::conv__foo" and the other is "aten__conv::foo" - feel it's pretty unlikely in reality.
Are '+' or '-' allowed in operator name? @smessmer

Reply:
changed from '_' to '-'